### DISCUSSION TITLE
Use newRelic (Vue3, Nuxt3, Webpack5)

### DISCUSSION THREAD

**Discussion by ChrisChristidis** [nuxt/nuxt#20096] [Questions]
Hello,

I added a new file newrelic.js to the root of the project.
I run npm install newrelic.
I added node -r newrelic newrelic.js to the start command of package.json.

As a result, the newrelic_agent.log is created, but the newrelic is not recording any activity.

Do you have any solution about this issue?

Thanks in advance.

**Comment by ronenteva** [comment]
Did you figure it out?

**Reply by ChrisChristidis** [reply]
I found this... As I use an iis server, I added "-r newrelic" in the arguments="" of <httpPlatform ...> element of web.config.
After that it started recording everything.
